aten/src/ATen/native/sparse/SparseCsrTensorMath.h

```diff
@@ -59,6 +62,51 @@ inline void _check_dim(const Tensor& self, int64_t target_dim, c10::string_view
       " instead.");
 }
 
+template <bool train>
+inline void check_sparse_mm_reduce_impl_inputs(
+    const Tensor& self,
+    const Tensor& grad_out,
+    const Tensor& other,
+    const Tensor& row_indices,
+    const Tensor& ccol_indices,
+    const Tensor& csr2csc) {
+  TORCH_CHECK(self.is_sparse_csr(),"Expected self to be sparse CSR tensor.");
+  TORCH_CHECK(self.dense_dim() == 0, "Expected non-hybrid self tensor.");
```

Comment:
Do you support batched dimensions?

Reply:
No, batched dimensions are not supported. Add checks to assert `self` is 2d matrix.